### ISSUE TITLE
Colorized MAN pages ie LESS_TERMCAP_so

### ISSUE THREAD

**Issue by dngray** [jan-warchol/selenized#49]
I decided to adapt [solarized-man.plugin.zsh](https://github.com/zlsun/solarized-man/blob/master/solarized-man.plugin.zsh) however I am having trouble with `LESS_TERMCAP_so` getting that to look like the solarized theme.

I read [this guide](https://boredzo.org/blog/archives/2016-08-15/colorized-man-pages-understood-and-customized) hoping it would help. Here's what I have so far:

```zsh
function man() {
    env \
        LESS_TERMCAP_mb=$(printf "\e[1;31m") \
        LESS_TERMCAP_md=$(printf "\e[1;31m") \
        LESS_TERMCAP_me=$(printf "\e[0m") \
        LESS_TERMCAP_se=$(printf "\e[0m") \
        LESS_TERMCAP_so=$(printf "\e[0;39;47m") \
        LESS_TERMCAP_ue=$(printf "\e[0m") \
        LESS_TERMCAP_us=$(printf "\e[4;32m") \
        PAGER="${commands[less]:-$PAGER}" \
        _NROFF_U=1 \
        GROFF_NO_SGR=1 \
        PATH=${HOME}/bin:${PATH} \
    man "$@"
}
```

![1565021052](https://user-images.githubusercontent.com/48640805/62478998-04833680-b79c-11e9-9a01-44be40dc6832.png) Termite is on the left with Solarized and urxvt is on the right with Selenized.

**Comment by dngray** [comment]
Alternatively using the [terminal codes](https://github.com/jan-warchol/selenized/blob/master/whats-wrong-with-solarized.md#problems-with-implementation) the nearest I could get was:

```zsh
LESS_TERMCAP_so=$(printf "\e[1;47;1;37m") \
```

Which resulted in:

![1565067478](https://user-images.githubusercontent.com/48640805/62512369-3e3d5700-b807-11e9-8423-c7385fd5109c.png)

**Comment by dngray** [comment]
```zsh
LESS_TERMCAP_so=$(printf "\e[30;39;100m") \
```

This looks nice too:

![1565069468](https://user-images.githubusercontent.com/48640805/62513470-b7d74400-b80b-11e9-8ef9-1acea447bd0b.png)

Curious to know what one you think looks best @jan-warchol

**Comment by dngray** [comment]
Looks okay with the other color schemes too:

![1565069905](https://user-images.githubusercontent.com/48640805/62513841-e1dd3600-b80c-11e9-8475-9d9b7ef85236.png)

I don't really know what I am doing with the codes, so if it's wrong please do tell me.

**Comment by jan-warchol** [comment]
Hello! Thanks for your interest, issues and pull requests - they are very welcome! :-) I'm super busy this week, but I hope to have time on Saturday, and I plan to get back to you.

**Comment by jan-warchol** [comment]
I've recalled that I have this snippet in my private configuraton:

```
# Configure escape sequences for less so that it will know how to display
# colors for man etc. See also:
# http://boredzo.org/blog/archives/2016-08-15/colorized-man-pages-understood-and-customized
# https://linuxtidbits.wordpress.com/2009/03/23/less-colors-for-man-pages/
# http://stackoverflow.com/questions/34265221/how-to-colorize-man-page-in-fish-shell
man() {
	env \
		LESS_TERMCAP_mb=$(printf "\e[1;37m") \
		LESS_TERMCAP_md=$(printf "\e[1;37m") \
		LESS_TERMCAP_me=$(printf "\e[0m") \
		LESS_TERMCAP_se=$(printf "\e[0m") \
		LESS_TERMCAP_ue=$(printf "\e[0m") \
			man "$@"
}
```

What do you think about the result, @dngray ? I like how it's not too flashy but still makes the items stand out.

**Comment by dngray** [comment]
I guess that works if you like it to be monochrome. Mine looks prettier!

Maybe we could have "selenized-man" and "selenized-man-mono", what do you think? one thing I did notice is the background for `LESS_TERMCAP_so` does not show up if in a tty, with my configuration, but does for yours.

**Comment by jan-warchol** [comment]
Yeah, perhaps we can have two variants. Another option would be to use a less contrasting colors, e.g. blue and/or cyan (I just think that red is too strong, giving sort of "in your face" impression).

One think I definitely prefer in your version is status bar. :-)

Side note: I'll be away until Monday.

**Comment by dngray** [comment]
> Another option would be to use a less contrasting colors, e.g. blue and/or cyan (I just think that red is too strong, giving sort of "in your face" impression).

What do you think of this?

![tty_blue](https://user-images.githubusercontent.com/48640805/63090910-40f92400-bf4c-11e9-890c-38f1cc14ce60.png)

Looks good in a tty too, `ter-132n` too. 

> One think I definitely prefer in your version is status bar. :-)
> 
> Side note: I'll be away until Monday.

That's okay.

**Comment by jan-warchol** [comment]
Blue version looks awesome!
Closed via #50. Thanks for your patience!
